etc/tendrl/monitoring-integration/graphite/carbon.conf.sample

```diff
@@ -75,7 +75,7 @@ MAX_UPDATES_PER_SECOND = 500
 # until such point as a subsequent metric is received and fits within the
 # defined rate limit. Setting this value high (like "inf" for infinity) will
 # cause carbon to create the files quickly but at the risk of increased I/O.
-MAX_CREATES_PER_MINUTE = 50
+MAX_CREATES_PER_MINUTE = 1000
```

Comment:
what about if we make the MAX_CREATES_PER_MINUTE to infinity. Anyways the file count will not be increased by time it will remain the same. So I think it better we handle all the situation.

Reply:
mm, then we can give inf, i think that would be better